pipelines/batch/src/main/java/org/openmrs/analytics/ConvertResourceFn.java

```diff
@@ -115,4 +134,20 @@ public void processElement(ProcessContext processContext)
     HapiRowDescriptor element = processContext.element();
     writeResource(element);
   }
+
+  private Resource createNewFhirResource(String resourceType) {
+    try {
+      return (Resource)
+          Class.forName("org.hl7.fhir.r4.model." + resourceType).getConstructor().newInstance();
```

Comment:
should this be conditional? what about STU3? will it work irrespective?

Reply:
Handled the STU3 and other FHIR versions as well.